src/modules/minigame/water_rowing_rapids/rock/rock_01.tscn

```diff
@@ -1,6 +1,6 @@
-[gd_scene load_steps=12 format=3 uid="uid://xxxxx"]
+[gd_scene load_steps=12 format=3 uid="uid://2wcf3ktin40u"]
 
-[ext_resource type="Script" uid="uid://mlcfb2pqu5j5" path="res://modules/minigame/water_rowing_rapids/rock/rock.gd" id="1_5xl68"]
+[ext_resource type="Script" path="res://modules/minigame/water_rowing_rapids/rock/rock.gd" id="1_5xl68"]
```

Comment:
this needs a UID though or every person's local build will keep generating new ones and show up in git diffs